breadcrumbs.dust

```diff
@@ -0,0 +1,11 @@
+<ol class="c-breadcrumbs {class}" role="navigation" data-adaptivejs-component="stencil-breadcrumbs">
+    {?element}
+        {element|s}
```

Comment:
I’m guessing this follows from what is in the Select component. For that component, I think it’s sensible to support the use case of providing the entire `<select>` markup, including its options, as a string. It’s not something that always benefits from being broken down and re-assembled, and in the case of Select, it’s not visible in the final rendered component anyway.

However, in this case I don’t think we want to support feeding the markup for all the items in as a single string. I can’t think of a likely use case for it, since “decoration” would be a more suitable approach if working directly with client markup. More on this below.